doc/userguide/rules/tls-keywords.rst

```diff
@@ -17,9 +17,22 @@ Examples::
 
 ``tls.cert_subject`` can be used as ``fast_pattern``.
 
-``tls.cert_subject`` replaces the previous keyword name: ``tls_cert_subject``. You may continue
-to use the previous name, but it's recommended that rules be converted to use
-the new name.
+tls.subject
+~~~~~~~~~~~
+
+Legacy keyword to match TLS/SSL certificate Subject field.
+
+example:
+
+::
+
+  tls.subject:"CN=*.googleusercontent.com"
+
+Case sensitive, can't use 'nocase'.
+
+**Note:** ``tls.cert_subject`` replaces the previous keyword names: ``tls_cert_subject`` and ``tls.subject``. 
+You may continue to use the previous names, but it's recommended that rules be converted to use
```

Comment:
Just "it's recommended that rules be converted to use the new keyword", drop the "you may" part